auth/src/main/java/com/firebase/ui/auth/ui/phone/CountryListSpinner.java

```diff
@@ -69,8 +73,101 @@ public CountryListSpinner(Context context, AttributeSet attrs, int defStyle) {
         mDialogPopup = new DialogPopup(mCountryListAdapter);
         mTextFormat = "%1$s  +%2$d";
         mSelectedCountryName = "";
+    }
+
+    public void init(Bundle params) {
+        if (params != null) {
+            List<CountryInfo> countries = getCountriesToDisplayInSpinner(params);
+            setCountriesToDisplay(countries);
+            setDefaultCountryForSpinner(countries);
+        }
+    }
+
+    private void getCountrySpinnerIsosFromParams(Bundle params) {
+        if (params != null) {
```

Comment:
Let's:
- rename this method `init...` since it returns void
- remove the null check
- add nullability annotations.

Reply:
Done.